src/schemas/json/pre-commit-config.json

```diff
@@ -387,22 +126,7 @@
         },
         "stages": {
           "$comment": "(optional) confines the hook to the commit, merge-commit, push, prepare-commit-msg, commit-msg, post-checkout, post-commit or manual stage. See https://pre-commit.com/#confining-hooks-to-run-at-certain-stages.",
-          "type": "array",
-          "items": {
-            "type": "string",
-            "enum": [
-              "commit",
-              "merge-commit",
-              "push",
-              "prepare-commit-msg",
-              "commit-msg",
-              "post-checkout",
-              "post-commit",
-              "post-merge",
-              "post-rewrite",
-              "manual"
-            ]
-          }
+          "$ref": "https://json.schemastore.org/pre-commit-hooks.json#/definitions/stages"
```

Comment:
Not sure how to pass tests for absolute references that are not uploaded yet.

Reply:
See updated [document](https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#ref-from-schema-xjson-to-schema-yjson) about reference